src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+// books.js
+
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.payload];
+    case REMOVE_BOOK:
+      return [...state, action.payload];
```

Comment:
- [ ] I think the `REMOVE_BOOK` action is actually adding the book again instead of removing it. Let's use the ES6 filter() method to create a new array that does not contain the book we want to remove from the store.

Once you have managed to display the list of books from the Redux Store in the UI you will know for certain if our `REMOVE_BOOK` action it's working as expected.

Reply:
Hi Daniel, The removing and adding of books is in the next project.